cartographer/mapping/2d/probability_grid_range_data_inserter_2d.h

```diff
@@ -51,6 +51,10 @@ class ProbabilityGridRangeDataInserter2D : public RangeDataInserterInterface {
                       GridInterface* grid) const override;
 
  private:
+  // For each ray in 'range_data', inserts hits and misses into
+  // 'probability_grid'. Hits are handled before misses.
+  void CastRays(const sensor::RangeData& range_data,
```

Comment:
can this be static?

Reply:
It depends on nonstatic members, therefore no.

Reply:
I would hide it in .cc file in anonymous namespace. You will have to pass more parameters `hit_table_`, `miss_table_`, `insert_free_space`.